spring-cloud-spanner-spring-data-r2dbc/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>spring-cloud-gcp</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>3.5.5-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>Spring Framework on Google Cloud Module - Cloud Spanner R2DBC Spring Data Dialect</name>
+  <artifactId>spring-cloud-spanner-spring-data-r2dbc</artifactId>
+
+  <properties>
+    <spring-data-r2dbc.version>1.5.13</spring-data-r2dbc.version>
+    <spring-test.version>5.3.28</spring-test.version>
+    <cloud-spanner-r2dbc.version>1.2.2</cloud-spanner-r2dbc.version>
+
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>cloud-spanner-r2dbc</artifactId>
+      <version>${cloud-spanner-r2dbc.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.data</groupId>
+      <artifactId>spring-data-r2dbc</artifactId>
+      <version>${spring-data-r2dbc.version}</version>
```

Comment:
Do you really need this version here? I believe the version will already be provided by spring boot bom we import in the parent pom.xml.

Reply:
yes, tested. It's getting the version from the BOM. removed it from here